ChainSafe/web3.unity
# Mint/Transfer 1155 fix

Problem: 
Transfer 1155 wasn't working as we didn't have a mint function meaning we had an insufficient balance and it would throw an error.

Fix:
Created a test contract with an open mint function and added this to the unsorted samples class.
Added a button in the samples scene under more examples to showcase the working results.
Updated sample main scene and necessary scripts.
Closes #681

To test:
1. Login and press the TransferErc1155 button 
2. Expected behavior: Transaction will throw an error due to insufficient balance.
3. Press the Mint1155 button to mint an NFT to your account.
4. Expected behavior: 1155 NFT mints to your account.
5. Press the TransferErc1155 button.
6. Expected behavior: 1155 NFT will transfer away from your account.

![image](https://github.com/ChainSafe/web3.unity/assets/57473220/2473b497-4243-4bd4-b445-860536841604)
